app/controllers/api/cookies_consent_controller.rb

```diff
@@ -0,0 +1,23 @@
+# require 'ofn_web'
+
+module Api
+  class CookiesConsentController < BaseController
```

Comment:
IMO this belongs to the engine and therefore it should in its folder. Also, if we do that, this controller should be namespaced as part of the engine under the `Web` module.

Reply:
Agreed, I've moved all engine stuff to a different branch, Ill do that there. Thanks.